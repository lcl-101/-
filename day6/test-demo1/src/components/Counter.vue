<template>
  <div>
    {{ count }}
    <button @click="count++">count++</button>
  </div>
</template>

<script>
export default {
  data() {
    return {
      count: 0,
    };
  },
};
</script>
