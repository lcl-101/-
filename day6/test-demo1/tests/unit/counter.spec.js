import { mount } from "@vue/test-utils";
import Counter from "@/components/Counter.vue";

describe("Counter.vue", () => {
  const wrapper = mount(Counter);
  it("renders props.msg when passed", () => {
    expect(wrapper.text()).toMatchSnapshot();
  });
  it("cont++", () => {
    wrapper.find("button").trigger("click");
    expect(wrapper.vm.count).toBe(1);
  });
});
