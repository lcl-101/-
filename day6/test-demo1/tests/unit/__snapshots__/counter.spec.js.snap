// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter.vue renders props.msg when passed 1`] = `
"0
  count++"
`;
